src/systems/ec/implementations/react/packages/checkbox/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "private": true,
+  "name": "@ecl/ec-react-component-checkbox",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "2.0.0",
+  "description": "ECL EC React Checkbox",
+  "dependencies": {
+    "@ecl/ec-base": "^2.0.0",
+    "@ecl/ec-react-component-icon": "^2.0.0",
+    "@ecl/ec-component-form-feedback-message": "^2.0.0",
+    "@ecl/ec-component-form-form-group": "^2.0.0",
+    "@ecl/ec-component-form-help-block": "^2.0.0",
+    "@ecl/ec-component-form-label": "^2.0.0",
+    "@ecl/ec-component-form-checkbox": "^2.0.0"
```

Comment:
`"@ecl/ec-component-form-checkbox": "^2.0.0"` is actually a `devDependencies` and `"@ecl/ec-react-component-icon": "^2.0.0",` is the only `dependencies`. All the other dependencies can be removed. Please see https://github.com/ec-europa/europa-component-library/blob/v2/src/systems/ec/implementations/react/packages/button/package.json 

PS: this is not a blocker since the React component is not published, but it's cleaner to make it right

Reply:
Actually, if we want to be consistent, you should follow this example: https://github.com/ec-europa/europa-component-library/blob/v2/src/systems/ec/implementations/react/packages/text-input/package.json

Except that in your case, you have `"@ecl/ec-react-component-icon": "^2.0.0"` in your  `dependencies`. All the `@ecl/ec-component-*` dependencies are `devDependencies`. You don't need to depend on `@ecl/ec-base`